pkg/sfu/forwarder.go

```diff
@@ -228,6 +216,23 @@ func NewForwarder(codec webrtc.RTPCodecCapability, kind webrtc.RTPCodecType, log
 	return f
 }
 
+func (f *Forwarder) DetermineCodec(codec webrtc.RTPCodecCapability) {
+	if f.codec.MimeType != "" {
+		return
+	}
```

Comment:
MimeType for SimulcastCodec is empty?

Reply:
Now the downtrack's codec is determined when it is binded(since we need to negotiate with client to decide which codec should be use), but the forwarder is created in the downtrack's constructor, so add a function `DetermineCodec` to set the codec that be actually use. This check is avoid the function be called twice. Trying to move NewForwarder to Downtrack's Bind method, but found I need to check forwarder is not nil in everywhere, so keep it in downtrack's constructor

Reply:
Got it.